enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -0,0 +1,353 @@
+It defines a set of CPUs that can be allocated to guaranteed pods/containers that still require
+non-exclusive ones.
+
+The CPU set must not overlap with `spec.cpu.reserved` or `spec.cpu.isolated` and NTO
+should return an error in case it does.
+
+The reason for using an annotation and not new API field 
+is because there is a coming feature named shared-partition
+that uses similar convention i.e., shared cpus, and would be 
+enabled via NTO as well.
+
+Once shared-partition lands, 
+the annotation should be [deprecated](mixed-cpu-node-plugin.md#removing-a-deprecated-feature)
+and node-plugin will use the shared cpus specified 
+at `spec.cpu.shared`.
+
+Both workloadHints and the annotation have to be specified in order to activate the feature.
+If only one of them would be specified, NTO should report a warning (in logs or status). 
+
+PerformanceProfile example:
+```yaml
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: example-performance-profile
+  annotations:
+    mixed-cpu.openshift.io/shared-cpus: "2-3"
+spec:
+  cpu:
+    reserved: "0-1"
+    isolated: "4-8"
+  hugepages:
+    defaultHugepagesSize: "1G"
+    pages:
+      - size: "1G"
+        count: 2
+        node: 0
+  realTimeKernel:
+    enabled: true
+  workloadHints:
+    mixedCpus: true
+  nodeSelector:
+    node-role.kubernetes.io/performance: "test"
+```
+
+Specify the annotation and `mixedCpus: true` in `workloadHints` activates the feature,
+signals NTO to update the `reservedSystemCpus` in Kubelet config,
+reboots the nodes that are associated with the updated PerformanceProfile,
+and deploys the mixed-cpu-node-plugin.
+The logic of updating reserved-cpus pool is NTO's responsibility and would be implemented as part of this feature.
+
+Remove the annotation or `mixedCpus: true` from `workloadHints` reverts the changes from Kubelet `reservedSystemCpus`,
+reboots the nodes that are associated with the updated PerformanceProfile,
+and deletes the mixed-cpu-node-plugin from the cluster.
+
+The feature can be activated/deactivated on a running system.
+
+NOTE: After shared-partition lands, the shared cpus will be allocated from `spec.cpu.shared` cpuset
+and no longer be allocated from Kubelet `reservedSystemCpus` cpuset.
+
+### Workflow Description
+
+The premise is that OCP cluster is running, and there's an active performance-profile that already tuned the system.
+
+1. The cluster administrator wants to support both exclusive and shared cpus for workloads running on the cluster.
+2. The cluster administrator specifies the shared CPU ids via annotation in the PerformanceProfile.
+3. The cluster administrator waits for MCO to kick in, update and reboot the nodes.
+4. The cluster administrator waits for the node to come back from reboot.
+5. The application administrator wants to deploy its DPDK application as a Guaranteed pod with shared CPUs.
```

Comment:
nit: idiomatically, we'd use `their` here instead of `its` as we're talking about a person, and typically the gender neutral pronouns we use there are conjugations of they

Reply:
Done. thanks